controllers/users.controllers.js

```diff
@@ -1,5 +1,10 @@
-const { fetchAllUsers, removeUserById } = require('../models/users.models');
+const {
+  fetchAllUsers,
+  removeUserById,
+  addUser
+} = require('../models/users.models');
 
+// GET
 const getAllUsers = (req, res, next) => {
   fetchAllUsers()
     .then((users) => {
```

Comment:
need to at some point decide on convention for 200 status codes, whether to implicitly use res.status(200) or leave it out completely

Reply:
I've made another commit so that all controllers send status code explicitly